server/utils.go

```diff
@@ -103,6 +103,20 @@ func decrypt(key []byte, text string) (string, error) {
 	return string(unpadMsg), nil
 }
 
+func parseOwner(full, baseURL string) (string, string) {
```

Comment:
1/5 would prefer `parseOrg[anization]` as a more intention-revealing name.

Reply:
@levb definitely agree with you here